wire/wire.proto

```diff
@@ -0,0 +1,13 @@
+syntax = "proto3";
+package basictracer_go.wire;
+option go_package = "wire";
+
+message TracerState {
+  int64 trace_id = 1;
```

Comment:
nit: `fixed64` will be better here and on the line below. (["Always eight bytes. More efficient than uint64 if values are often greater than 256."](https://developers.google.com/protocol-buffers/docs/proto3#scalar))